io-engine/src/subsys/config/opts.rs

```diff
@@ -87,6 +87,13 @@ impl From<NvmfTgtConfig> for Box<spdk_nvmf_target_opts> {
         let mut out = Self::default();
         copy_str_with_null(&o.name, &mut out.name);
         out.max_subsystems = o.max_namespaces;
+        let c = try_from_env("NVMF_TGT_CRDT", 0);
```

Comment:
Should we enable it by default?
Also, can we make it a cli argument which would be enabled via helm?

Reply:
Fixed